multiqc/modules/longranger/longranger.py

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from Longranger """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re
+import os
+import csv
+
+from multiqc import config
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import table
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    """ Longranger module """
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Longranger', anchor='longranger',
+        href="https://www.10xgenomics.com/",
+        info="A set of analysis pipelines that perform sample demultiplexing, "
+        "barcode processing, alignment, quality control, variant calling, phasing, "
+        "and structural variant calling.")
+
+        self.headers = OrderedDict()
+        self.headers['longranger_version'] = {
```

Comment:
Could you instead add this to the `info` string, if all values are the same? Then perhaps only add this column if they differ.

Reply:
I'm not sure I understand. Can you give an example of a module that does this?

Reply:
```python
longranger_versions = set([d['longranger_version'] for d in self.longranger_data])
if len(longranger_versions) == 1:
    infotext += "All samples were processed with Longranger version {}".format(longranger_versions[0])
else:
    self.headers['longranger_version'] = {
    #...
```